src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,3 +1,33 @@
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import React from 'react';
+import './PostInfo.scss';
+import { Post } from '../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      {' Posted by  '}
+
+      {post.user && (
+        <UserInfo user={post.user} />
+      )}
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+
+    <hr />
+
```

Comment:
As far as I noticed from the original layout there is no need to use this <hr /> for cases when we have comments